cmd/main.go

```diff
@@ -244,6 +244,14 @@ func electMaster(mgr manager.Manager, nsxClient *nsx.Client) {
 	log.Info("I'm trying to be elected as master")
 	<-mgr.Elected()
 	log.Info("I'm the master now")
+	// In HA mode, there can be a brief period where both the old and new leader
+	// operators are active simultaneously. This occurs when the new operator
+	// acquires the lease before the old operator recognizes it has lost the lease,
+	// leading to a potential race condition. To mitigate this, the new master
+	// operator is configured to wait for 15 seconds, which is the default Leader
+	// Election Lease Duration, ensuring a smooth transition.
+	log.Info("waiting a 15-second delay to let the old instance know that it has lost its lease")
```

Comment:
Could we wait for 18 or 20 seconds to ensure old leader gone?

Reply:
Update comments as discussed offline, thanks